src/core/electrostatics_magnetostatics/magnetic_non_p3m_methods.cpp

```diff
@@ -389,6 +389,7 @@ double magnetic_dipolar_direct_sum_calculations(int force_flag,
 
 int dawaanr_set_params() {
   if (n_nodes > 1) {
+    runtimeErrorMsg() << "MPI parallelization not supported by DAWAANR.";
```

Comment:
Please adjsut the method names in the error message to mirror the Python class names

Reply:
Why? This core function is unaware of the Python interface. When this message is sent, Python displays it at the end of the traceback as `Exception: Could not activate magnetostatics method DipolarDirectSumCpu: ERROR: MPI parallelization not supported by DAWAANR.` This is done by:
https://github.com/espressomd/espresso/blob/76be86ca7678afbf302805ed5dcaf57137e89569/src/python/espressomd/magnetostatics.pyx#L273-L275

Reply:
If your real concern is that the `DAWAANR` abbreviation is irrelevant to the user, we could just remove the `"by DAWAANR"` at the end of the error message.

Reply:
I think I get it: the core function prefix `dawaanr_` is a C-style namespace, right? And DAWAANR was renamed into DipolarDirectSum in the Python interface for convenience. If that's the case, removing `"by DAWAANR"` from the message would be sufficient in my opinion, and will avoid any confusion if we decide to rename magnetostatics Python classes in the future.